apps/web/app/[orgShortCode]/settings/SettingsSidebar.tsx

```diff
@@ -1,11 +1,210 @@
 'use client';
 
-import { Flex } from '@radix-ui/themes';
+import React from 'react';
+import { usePathname } from 'next/navigation';
+import { Flex, Text } from '@radix-ui/themes';
+import Link from 'next/link';
+import { cn } from '@/lib/utils';
+import { useGlobalStore } from '@/providers/global-store-provider';
+import {
+  CircleUser,
+  Mail,
+  Lock,
+  Building2,
+  CreditCard,
+  AtSign,
+  Globe,
+  Users,
+  UserRoundPlus,
+  HeartHandshake
+} from 'lucide-react';
+
+interface links {
+  label: string;
+  to: string;
+  icon: React.ReactNode;
+}
 
 export default function SettingsSidebar() {
+  const orgShortCode = useGlobalStore((state) => state.currentOrg.shortCode);
+  const pathname = usePathname();
+
+  const personalLinks: links[] = [
+    {
+      label: 'Profile',
+      to: `/${orgShortCode}/settings/user/profile`,
+      icon: <CircleUser />
+    },
+    {
+      label: 'Personal Addresses',
+      to: `/${orgShortCode}/settings/user/addresses`,
+      icon: <Mail />
+    },
+    {
+      label: 'Security',
+      to: `/${orgShortCode}/settings/user/security`,
+      icon: <Lock />
+    }
+  ];
+  const orgSetupLinks = [
+    {
+      label: 'Org Profile',
+      to: `/${orgShortCode}/settings/org`,
+      icon: <Building2 />
+    },
+    {
+      label: 'Billing',
+      to: `/${orgShortCode}/settings/org/setup/billing`,
+      icon: <CreditCard />
+    }
+  ];
+
+  const orgUserLinks: links[] = [
+    {
+      label: 'Members',
+      to: `/${orgShortCode}/settings/org/users/members`,
+      icon: <Users />
+    },
+    {
+      label: 'Invites',
+      to: `/${orgShortCode}/settings/org/users/invites`,
+      icon: <UserRoundPlus />
+    },
+    {
+      label: 'Teams',
+      to: `/${orgShortCode}/settings/org/users/teams`,
+      icon: <HeartHandshake />
+    }
+  ];
+  const orgMailLinks: links[] = [
```

Comment:
```suggestion
  const orgMailLinks: NavLinks[] = [
```

Reply:
okk, should I do NavLinks or settingLinks?

Reply:
anything you like just call it like `CapitalName`